lib/test/unit/test-suite-runner.rb

```diff
@@ -7,14 +7,15 @@
 # License:: Ruby license.
 
 require "etc"
+require_relative "test-run-context"
```

Comment:
Could you add an empty line between external libraries block and internal files block? 

```suggestion

require_relative "test-run-context"
```

Reply:
Sure!